stdlib/html/parser.pyi

```diff
@@ -28,3 +29,17 @@ class HTMLParser(ParserBase):
     def parse_pi(self, i: int) -> int: ...  # undocumented
     def parse_starttag(self, i: int) -> int: ...  # undocumented
     def set_cdata_mode(self, elem: str) -> None: ...  # undocumented
+    def parse_comment(self, i: int, report: int = ...) -> int: ...  # undocumented
+    def parse_declaration(self, i: int) -> int: ...  # undocumented
+    def parse_marked_section(self, i: int, report: int = ...) -> int: ...  # undocumented
+    def updatepos(self, i: int, j: int) -> int: ...  # undocumented
+    rawdata: str  # undocumented
+    cdata_elem: str | None  # undocumented
+    convert_charrefs: bool  # undocumented
+    interesting: Pattern[str]  # undocumented
+    lasttag: str  # undocumented
+    lineno: int  # undocumented
+    offset: int  # undocumented
```

Comment:
These seem to be local variables inside `parse_starttag()`, not set on the instance.

```suggestion
```

Reply:
Actually these should be moved to `ParserBase`.

Reply:
@Akuli
If you instatiate `HTMLParser` it calls `self.reset()`, which in turn calls `_markupbase.ParserBase.reset(self)` and `_markupbase.ParserBase.reset(self)` sets these two attributes on the instance:

![image](https://user-images.githubusercontent.com/80432516/147128364-e2b06aa2-48ce-45ad-83ad-0a4b712ef7c5.png)

![image](https://user-images.githubusercontent.com/80432516/147128232-4c8be25f-42ab-4b82-b8c2-73c11f09daf6.png)

and since `ParserBase` can't be instantiated but a user could still access these attributes on `HTMLParser` it made sense to me to put them here. But happy to remove if you want?

Reply:
![image](https://user-images.githubusercontent.com/80432516/147128808-0d2e43f1-ce37-4472-b67f-4d9fac138755.png)

Reply:
It seems to me that you're supposed to call `.reset()` before doing anything with the `ParserBase` instance. But this also doesn't matter much, because `ParserBase` is apparently not used for anything else than `HTMLParser`.

Reply:
> Actually these should be moved to `ParserBase`.

@Akuli apologies didn't see this before my original comment, I can move them now 👍